src/targets/gpu/lowering.cpp

```diff
@@ -379,28 +336,26 @@ struct miopen_apply
         });
     }
 
-    template <class T>
-    void add_generic_op(std::string name)
+    void add_generic_op(const std::string& op_name, const std::string& gpu_name)
```

Comment:
Rather than pass in the `gpu_name` to the function, can we just prepend `"gpu::"` to `op_name` in L346?

Reply:
Well, I dont know if all operators in the future will follow that pattern. We could probably refactor this in the future if we think we wont break this convention.

Reply:
I think it would be easier to change it now since I can't really think of any operator where it would differ. If in the future we need to have a different name, it might be easier to just have the `gpu_name` be an optional parameter.